document-store/src/main/java/org/hypertrace/core/documentstore/expression/operators/AggregationOperator.java

```diff
@@ -3,6 +3,8 @@
 public enum AggregationOperator {
   AVG,
   COUNT,
+  DISTINCT,
```

Comment:
what difference between `DISTINCT` and `DISTINCT_Count`?

Reply:
`DISTINCT` will fetch the distinct values. `DISTINCT_COUNT` will get the number of distinct values.
In other words, `DISTINCT` returns a Set where `DISTINCT_COUNT` returns the size of that set.